src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -857,6 +857,17 @@ public void switchFolder(FolderElement switchTo) {
 
 		upFolder.setEnabled(!currentFolder.isRoot());
 	}
+	
+	private void filter(String text) {
+		String filter = "f:" + text + " ";
+		boolean isMatch = Pattern.matches(".*f:" + text + " .*", search.getText());
+		if (isMatch == false) {

```

Comment:
Put isMatch inside if, and use else then, not else if as this really is not needed.

Reply:
This is not resolved.

Reply:
i did use else and have isMatch == false in the if

Reply:
or you meant something else for the "isMatch inside if"

Reply:
> Put isMatch inside if

Instead of doing this : 
boolean isMath = Pattern.matches(".*f:" + text + " .*", search.getText());
if (isMath == false) {
...
}

You should do this :
if (!Pattern.matches(".*f:" + text + " .*", search.getText())) {
...
}

Reply:
ahhh, i see, Thanks Max!